src/rpdk/core/contract/suite/handler_update.py

```diff
@@ -46,5 +50,47 @@ def contract_update_list_success(updated_resource, resource_client):
     # should be able to use the created model
     # to read since physical resource id is immutable
     _create_request, _created_model, _update_request, updated_model = updated_resource
+    resource_client.assert_primary_identifier_not_updated(
+        resource_client.primary_identifier_paths, _created_model, updated_model
+    )
     models = test_list_success(resource_client, updated_model)
     assert updated_model in models
+
+
+@pytest.mark.update
+def contract_update_create_only_property(resource_client):
+
+    if resource_client.create_only_paths:
```

Comment:
Use pytest.skip here to denote the test didnt run: https://github.com/aws-cloudformation/cloudformation-cli/blob/f12d5b8a5d75efb71f9d259ab045020ef79fcbb0/src/rpdk/core/contract/suite/handler_delete.py#L94